### PR TITLE
Update gcc to 11

### PR DESCRIPTION
**Pull request by az1919**

### CODE REVIEW

File: toolchains/compilers/gcc_arm_none_eabi/impl/gcc_arm_none_versions.bzl [hunk 1]
```diff
@@ -51,8 +51,40 @@ _PLATFORM_SPECIFIC_CONFIGS_9 = {
...
+    },
+    "windows": _WINDOWS_11,
+    "windows server 2019": _WINDOWS_11,
+    "windows 10": _WINDOWS_11,
+    "mac os x": {
+        "full_version": "11.3.rel1",
+        "remote_compiler": {
+            "url": "https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu/11.3.rel1/binrel/arm-gnu-toolchain-11.3.rel1-darwin-x86_64-arm-none-eabi.tar.xz",
+            "sha256": "826353d45e7fbaa9b87c514e7c758a82f349cb7fc3fd949423687671539b29cf",
+            "strip_prefix": "arm-gnu-toolchain-11.3.rel1-x86_64-arm-none-eabi",
```
Comment (silvergasp) [inline toolchains/compilers/gcc_arm_none_eabi/impl/gcc_arm_none_versions.bzl:80]: I think this needs to be "arm-gnu-toolchain-11.3.rel1-darwin-x86_64-arm-none-eabi" the prefix is different for each platform.

See the build log here https://github.com/bazelembedded/bazel-embedded/actions/runs/3335309869/jobs/5519150813

**Review by silvergasp — CHANGES_REQUESTED** [review]
Thanks for getting back to me with another PR so soon. Sorry I had to revert the origional. There was a couple other issues with the CI that are now fixed, but somehow I let it slip through.

File: toolchains/compilers/gcc_arm_none_eabi/impl/gcc_arm_none_versions.bzl (hunk 1 above)
Comment (az1919) [reply toolchains/compilers/gcc_arm_none_eabi/impl/gcc_arm_none_versions.bzl]: Hi
Thanks for the review, I have corrected the strip_prefix for darwin platform

**Comment by silvergasp** [comment]
Thanks, that seems to be sorted now!

**Pull request merged**